### ISSUE TITLE
Exercise 3: learning_rate parameter

### ISSUE THREAD

**Issue by ivanrvpereira** [LDSSA/batch2-blu12#5]
I think there is a bug in the exercise description.
`* With a learning rate of 0.0005` should be `* With a learning rate of 0.00005` that is equal to 5e-05 as in the assert
